src/register/data/reducers.js

```diff
@@ -108,6 +110,7 @@ const reducer = (state = defaultState, action = {}) => {
         return {
           ...state,
           backendCountryCode: countryCode,
+          IPBasedCountryCode: countryCode,
```

Comment:
why can't we use `backendCountryCode`? Why do we need a new state for it?

Reply:
this ```backendCountryCode``` is reset when ```BACKUP_REGISTRATION_DATA.BEGIN``` is fired so i did not touch that state I created new one which does not get reset when above action is fired.